src/main/java/com/synopsys/integration/blackduck/nexus3/task/inspector/model/TemporaryOriginView.java

```diff
@@ -0,0 +1,37 @@
+package com.synopsys.integration.blackduck.nexus3.task.inspector.model;
+
+import java.util.Date;
+
+import com.synopsys.integration.blackduck.api.core.BlackDuckView;
+import com.synopsys.integration.blackduck.api.generated.enumeration.ComponentSourceType;
+
+public class TemporaryOriginView extends BlackDuckView {
```

Comment:
Is this to fulfill a requirement that should be in our library that doesn't yet exist?

Reply:
Yes the current OriginView in our library does not have the correct fields so the deserialization fails. There are tickets to correct this so we won't need this in the future.